aio/src/app/layout/mode-banner/mode-banner.component.ts

```diff
@@ -4,11 +4,15 @@ import { VersionInfo } from 'app/navigation/navigation.service';
 @Component({
   selector: 'aio-mode-banner',
   template: `
-  <div *ngIf="mode === 'archive'" class="mode-banner alert archive-warning">
-    <p>This is the <strong>archived documentation for Angular v{{version?.major}}.</strong>
-    Please visit <a href="https://angular.io/">angular.io</a> to see documentation for the current version of Angular.</p>
-  </div>
-  `
+    <div *ngIf="mode === 'archive'" class="mode-banner alert archive-warning">
+      <p>
+        This is the
+        <strong>archived documentation for Angular v{{ version?.major }}.</strong> Please visit
+        <a href="https://angular.io/">angular.io</a> to see documentation for the current version of
+        Angular.
+      </p>
```

Comment:
Why not:

```suggestion
      <p>
        This is the <strong>archived documentation for Angular v{{ version?.major }}.</strong> Please visit
        <a href="https://angular.io/">angular.io</a> to see documentation for the current version of Angular.
      </p>
```

That seems to be enough to satisfy the currently set limit of 120 characters.